### PR TITLE
Change position of tabbar badges

### LINKED ISSUE #2432: [Enhancement] Tabbar with medium text badge
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
DRB is working to enhance the tabbar to make is possible to see the number of unread messages. A medium badge with text is used to show this info. The placement and alignment of the badge with text is not correct.

## Describe the solution you'd like
Adjust the position and the alignment (left-align) of the badge.

## Have you considered any alternatives?
No

## Are there any additional context?

Text-badge-layout-issue

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [x] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by mictro**
## Which issue does this PR close?

This PR closes #2432 

## What is the new behavior?
Reposition tab-bar badges in accordance with updated design. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [X] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [X] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [X] Request that the changes are code-reviewed 
- [X] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

**Review by jkaltoft — CHANGES_REQUESTED** [review]
I really like the explanation(s) for why we're using magic numbers. Much better than before 👍🏻

I have a few requests for change. Otherwise great work.

File: libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss [hunk 1]
```diff
@@ -41,16 +46,34 @@ ion-tab-button {
...
+    --kirby-badge-left: 1.6em;
+
+    @include utils.slotted(kirby-badge) {
+      &.md {
+        --kirby-badge-position: relative;
+        --kirby-badge-top: 0;
+        --kirby-badge-left: 0;
+
+        margin-left: utils.size('xxxxs');
+        margin-bottom: calc(utils.size('xxxxs') / 2); // 799 -> 798
```
[the remark this answers is not in the document]
Comment (mictro) [reply libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss:69]: Good point - corrected

[the remark this answers is not in the document]
Comment (mictro) [reply libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss:69]: Work note removed

File: libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss [hunk 2]
```diff
@@ -41,16 +46,34 @@ ion-tab-button {
...
+    --kirby-badge-left: 1.6em;
+
+    @include utils.slotted(kirby-badge) {
+      &.md {
+        --kirby-badge-position: relative;
+        --kirby-badge-top: 0;
+        --kirby-badge-left: 0;
+
+        margin-left: utils.size('xxxxs');
+        margin-bottom: math.div(utils.size('xxxxs'), 2);
```
Comment (jkaltoft) [inline libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss:69]: Remember to import the math module (beginning of file):

```scss
@use 'sass:math';
```

Will fix https://github.com/kirbydesign/designsystem/actions/runs/3319146598/jobs/5483935259#step:4:36

**Review by jkaltoft — CHANGES_REQUESTED** [review]
There's a missing module import that makes the build fail. When that is fixed everything should be ready to go 🙂

File: libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss (hunk 2 above)
Comment (mictro) [reply libs/designsystem/src/lib/components/tabs/tab-button/tab-button.component.scss:71]: Bummer... - fixed

**Review by jkaltoft — APPROVED** [review]
🚀

**Pull request merged**